opencsp/common/lib/cv/fiducials/AbstractFiducials.py

```diff
@@ -102,9 +148,13 @@ def scale(self) -> list[float]:
         return ret
 
     def _render(self, axes: matplotlib.axes.Axes):
-        """
-        Called from render(). The parameters are always guaranteed to be set.
-        """
+        # Render the fiducial on the given axes.
```

Comment:
I hate that I can't get this documentation by hovering over it in vscode, just because the protected method will show up in the auto-generated documentation if this is kept in a docstring.

Is there any way that we can get the best of both worlds? Don't show the method in the auto-generated docs, but keep the documentation in a docstring so that I can get the description on a mouse hover?